draft-ietf-quic-transport.md

```diff
@@ -4086,6 +4086,7 @@ max_ack_delay (0x000b):
   delays in alarms firing.  For example, if a receiver sets a timer for 5ms
   and alarms commonly fire up to 1ms late, then it should send a max_ack_delay
   of 6ms.  If this value is absent, a default of 25 milliseconds is assumed.
+  Values of 2^14 or greater are invalid.
```

Comment:
Can you clarify what should happen if the value is invalid?  ie: Should the receiver cap the value at 2^14 or should the peer close the connection?

Reply:
I thought that we already had text to that effect (the adjacent parameters are similarly terse).  Added one.

Reply:
Thanks!